playground/assets/styles.css

```diff
@@ -0,0 +1,342 @@
+  flex-direction: column;
+  height: 100vh;
+  margin: 0;
+}
+
+#root,
+.playground-container {
+  height: 100%;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  height: 30px;
+  padding: 11px 27px;
+  background-color: #1a2b34;
+  color: #e0e0e0;
+  position: relative;
+  z-index: 20;
+}
+
+@media (max-width: 400px) {
+  header {
+    padding: 10px;
+  }
+}
+
+header a,
+header a:visited,
+header a:focus,
+header a:active,
+header a:hover {
+  color: #fff;
+  text-decoration: none;
+}
+
+.logo-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  height: 34px;
+  margin-right: 10px;
+}
+
+header h1 {
+  font-size: 20px;
+  display: block;
+  font-family: inherit;
+  font-weight: 400;
+  line-height: 18px;
+  position: relative;
+  z-index: 9999;
+  margin: 0;
+}
+
+#version {
+  font-size: 0.5em;
+  line-height: 0;
+  opacity: 0.5;
+}
+
+.links {
+  display: flex;
+  font-size: 16px;
+}
+.links > * + * {
+  margin-left: 15px;
+}
+
+.editors-container {
+  display: flex;
+  flex: 1;
+  min-height: 0;
+}
+
+.editors {
+  display: flex;
+  flex-flow: row wrap;
+  flex: 1;
+}
+
+.editor {
+  box-sizing: border-box;
+  display: flex;
+  flex: 1 1 100%;
+  position: relative;
+  border-bottom: 1px solid #ddd;
+}
+
+.disabled-output-panel {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 15px;
+  text-align: center;
+}
+
+.disabled-output-panel .explanation {
+  font-size: 16px;
+  max-width: 400px;
+  color: darkred;
+}
+
+.disabled-output-panel .btn {
+  font-size: 22px;
+  padding: 10px 20px;
+}
+
+/* display as 2x2 grid */
+@media (min-width: 800px) {
+  .options-container {
+    border-right: 0;
+  }
+
+  .editor {
+    flex-basis: 50%;
+    border-left: 1px solid #ddd;
+    margin-left: -1px;
+  }
+}
+
+/* display as four columns */
+@media (min-width: 1200px) {
+  .editor {
+    flex-basis: 25%;
+  }
+}
+
+.editor.loading {
+  opacity: 0;
+}
+
+textarea, .CodeMirror {
+  height: auto;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  line-height: 1.6;
+  font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
+}
+
+.bottom-bar {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bottom-bar-buttons {
+  z-index: 10;
+  padding: 8px;
+}
+
+.bottom-bar-buttons-center {
+  text-align: center;
+}
+
+.bottom-bar-buttons-right {
+  float: right;
+}
+
+.bottom-bar-buttons > * + * {
+  margin-left: 9px;
+}
+
+@media (max-width: 799px) {
+  .bottom-bar-buttons {
+    top: auto;
+    bottom: 100%;
+    padding: 2px;
+  }
+
+  .bottom-bar-buttons > * + * {
+    margin-left: 2px;
+  }
+}
+
+.options-container {
+  display: none;
+  box-sizing: border-box;
+  width: 225px;
+  overflow-y: auto;
+  border-bottom: 1px solid #ddd;
+  border-right: 1px solid #ddd;
+  flex: 0 1 auto;
```

Comment:
Personally, I'd add this bother to align the sidebar:
```suggestion
  flex: 0 1 auto;
  border-top: 1px solid;
```
Here's the difference:
<img width="80%" src="https://user-images.githubusercontent.com/1079279/186413366-29d865c6-c6d4-424b-bfca-bd38ee664dee.gif">

Reply:
Do you have 30/20 vision? :D Good eye!